config/badwords.exs

```diff
@@ -50,387 +50,236 @@ config :glimesh,
-    "bugger",
-    "bum",
-    "butt",
-    "buttplug",
-    "clitoris",
-    "cock",
-    "coon",
-    "crap",
-    "cunt",
-    "damn",
-    "dick",
-    "dildo",
-    "dyke",
-    "fag",
-    "feck",
-    "fellate",
-    "fellatio",
-    "felching",
-    "fuck",
-    "fudgepacker",
-    "fudge",
-    "packer",
-    "flange",
-    "goddamn",
-    "god",
-    "damn",
-    "hell",
-    "homo",
-    "jerk",
-    "jizz",
-    "knobend",
-    "knob",
-    "labia",
-    "muff",
-    "nigger",
-    "nigga",
-    "penis",
-    "piss",
-    "poop",
-    "prick",
-    "pube",
-    "pussy",
-    "queer",
-    "scrotum",
-    "sex",
-    "shit",
-    "sh1t",
-    "slut",
-    "smegma",
-    "spunk",
-    "tit",
-    "tosser",
-    "turd",
-    "twat",
-    "vagina",
-    "wank",
-    "whore",
-    "wtf"
+
+    "۞", "۩", "ஜ", "卐", "︻╦╤─", "@$$", "..l..", "..i..", "..I..", "..|..", "ahole", "amcik", "andskota", "anus", "arschloch", "arse", "ash0le","ash0les", #Hate Speech Present
+    "asholes", "ass", "ass monkey", "assface", "assh0le", "assh0lez", "asshole", "assholes", "assholz", "asshopper", 
+    "assjacker", "asslick", "asslicker", "assmonkey", "assmunch", "assmuncher", "assnigger", "asspirate", "assrammer",
+    "assshit", "assshole", "asssucker", "asswad", "asswipe", "axwound", "ayir", "azzhole", "b\!\+ch", "b\!tch", "b00bs",
+    "b17ch", "b1tch", "bassterds", "bastard", "bastard", "bastardz", "basterds", "basterdz", "basturds", "bi\+ch", "bi7ch",
+    "Biatch", "bitch", "bitches", "blow job", "blowjob", "boffing", "boiolas", "bollock", "boobs", "breasts", "buceta", "butt"
+    "butt-pirate", "butthole", "buttlicker", "buttwipe", "c0ck", "c0cks", "c0k", "cabron", "caca", "carpetmuncher", "carpetmuncha", #particularly nasty ones
+    "carpet muncher", "cawk", "cawks", "cazzo", "chink", "chraa", "chuj", "cipa" "clit", "clits", "cnts", "cntz", "cock",
+    "cock-head", "cock-sucker", "cockhead", "cocks", "cocksucker", "crap", "cum", "cunt", "cunts", "cunty", "cuntz", "d4mn", "dab",
+    "damn", "daygo", "dego", "dick", "dicksucker", "dicklicker", "dickfucker", "dike", "dild0", "dild0s", "dildo", "dildos", "dilld0",
+    "dilld0s", "dirsa", "dominatricks", "dominatrics", "dominatrix", "dupa", "dyke", "dziwka", "ejackulate", "ejakulate", "Ekrem",
+    "ekto", "enculer", "enema", "f u c k", "f u c k e r", "faen", "fag", "fag1t", "faget", "fagg1t", "faggit", "faggot", "fagit", "fags", #Hate Speech Present
+    "fagz", "faig", "faigs", "fanculo", "fanny", "fart", "fatass", "fcuk", "feces" "feg", "Felcher", "ficken", "fiddler", "fitt", 
+    "flikker", "flipping the bird", "fokken", "foreskin", "fotze", "fu\(", "fuck", "fucker", "fuckin", "fucking", "fucks","fudge packer",
+    "fuk", "Fukah", "fuken", "fuker", "fukin", "fukk", "fukkah", "fukken", "fukker", "fukkin", "futkretzn", "fux0r", "g00k", 
+    "gay","gayboy","gayboi","gaygirl", "gaygurl", "gays", "gayz", "god-damned", "gook", "guiena", "h00r", "h0ar", "h0r", "h0re", "h4x0r", #Hate Speech Present
```

Comment:
I'm not sure `gay` and `lesbian` should be considered bad words. I understand they can be used in offensive contexts, but the words themselves are not bad.

Reply:
There are also a lot of other words in here that aren't bad in them self but also dependent on the context. Possible solution would be to split it up into age categories as some of them allows for more lenient usage of these words than others and based on that filter them out.